eslint/babel-eslint-plugin/src/rules/no-invalid-this.cjs

```diff
@@ -2,7 +2,11 @@ const ruleComposer = require("eslint-rule-composer");
 const eslint = require("eslint");
 const eslintVersion = eslint.ESLint.version;
 
-const noInvalidThisRule = new eslint.Linter().getRules().get("no-invalid-this");
+const noInvalidThisRule = (
+  parseInt(eslintVersion, 10) >= 9
+    ? require("eslint/use-at-your-own-risk").builtinRules
```

Comment:
https://github.com/eslint/eslint/issues/15546#issuecomment-1023810087

Reply:
See also ESLint's migration guide: https://eslint.org/docs/latest/use/migrate-to-8.0.0#-the-lib-entrypoint-has-been-removed.